core/src/main/scala/org/bitcoins/core/script/arithmetic/ArithmeticOperations.scala

```diff
@@ -4,148 +4,148 @@ import org.bitcoins.core.script.ScriptOperationFactory
 import org.bitcoins.core.script.constant.ScriptOperation
 
 /** Created by chris on 1/6/16. */
-sealed trait ArithmeticOperation extends ScriptOperation
+sealed abstract class ArithmeticOperation extends ScriptOperation
```

Comment:
This is for Java interop?

Reply:
Yes -- for the `scalac` 2.11.x -- but apparently for `scalac`  2.12.x this doesn't matter any more https://www.scala-lang.org/news/2.12.0/